src/verse.cpp

```diff
@@ -175,7 +175,7 @@ int Verse::AdjustSylSpacing(FunctorParams *functorParams)
 
     // Use the syl because the content bounding box of the verse might be invalid at this stage
     int overlap = params->m_lastSyl->GetContentRight() - (firstSyl->GetContentLeft() + xShift);
-    overlap += lastSyl->CalcConnectorSpacing(params->m_doc, params->m_staffSize);
+    overlap += params->m_lastSyl->CalcConnectorSpacing(params->m_doc, params->m_staffSize);
```

Comment:
What is the reason for this change? Was it wrong?

Reply:
Logical. lastSyl is current last syllable. But in this line we calculate space between current syllable and previous syllable. This fix is not related to the current task.

Reply:
Correct. Leave it